### PR TITLE
feat(PDEV-3187): Update Raml for Put Method

### PR DESCRIPTION
**Pull request by junior-blx**
related to https://github.com/trizic/platform/pull/2050

### CODE REVIEW

File: api/v10/index.raml [hunk 1]
```diff
@@ -276,6 +276,29 @@
               401:
                 description: |
                   Request is not authorized.
+          put:
+            description: |
+              ### Submit completed account opening Forms and initiate the account opening update workflow
```
Comment (jamesdaniello) [inline api/v10/index.raml]:
```suggestion
              ### Submit an update to an existing account opening form
```

File: api/v4/account/opening.raml [hunk 2]
```diff
@@ -22,6 +22,29 @@ post:
         application/json:
           schema: !include ../../common/schema/generic.res.json
           example: !include ../../common/examples/generic.res.json
+put:
+  is:
+    - tokenized
+    - expiry
+
+  description: |
+    ### Initiate an Update Opening process through Account Opening service
```
Comment (jamesdaniello) [inline api/v4/account/opening.raml]:
```suggestion
    ### Update an existing Account Opening Form
```

File: api/v10/index.raml (hunk 1 above)
Comment (junior-blx) [reply api/v10/index.raml]: updated

File: api/v4/account/opening.raml (hunk 2 above)
Comment (junior-blx) [reply api/v4/account/opening.raml]: updated

**Pull request merged**